airflow/providers/google/cloud/hooks/bigtable.py

```diff
@@ -248,7 +248,9 @@ def delete_table(self, instance_id: str, table_id: str, project_id: str) -> None
         """
         instance = self.get_instance(instance_id=instance_id, project_id=project_id)
         if instance is None:
-            raise RuntimeError("Instance %s did not exist; unable to delete table %s" % instance_id, table_id)
+            raise RuntimeError(
+                "Instance {} did not exist; unable to delete table {}".format(*instance_id), table_id
```

Comment:
Should we just use f-strings here instead?

Reply:
Why not

Reply:
(this has been automatically converted to riff, so this is interesting change.

Reply:
It lookes like Ruff treated table_id as linked exception :)

Reply:
AH.... That WAS a bug :)

It should have been:

```
raise RuntimeError("Instance %s did not exist; unable to delete table %s" % (instance_id, table_id))
```